core/templates/pages/exploration-editor-page/editor-tab/graph-directives/state-graph-visualization.directive.html

```diff
@@ -62,12 +62,42 @@
     right: 2px;
     top: 2px;
   }
+
+  .oppia-graph-resize-button-extra {
+    font-size: larger;
+  }
+
+  .oppia-node-label-x {
+    fill: gray;
```

Comment:
Please use the hex equivalent for the color here and below (see [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css)).